src/aleph/netty.clj

```diff
@@ -1226,33 +1248,49 @@ initialize an DnsAddressResolverGroup instance.
               (let [ch (.channel ^ChannelFuture f)]
                 (maybe-ssl-handshake-future ch)))))))))
 
+(defn- add-ssl-handler
+  [pipeline-builder ssl-ctx]
+  (fn [^ChannelPipeline pipeline]
+    (append-handler-to-pipeline
+     pipeline "ssl-hanlder"
+     (.newHandler ^SslContext ssl-ctx (-> pipeline .channel .alloc)))
+    (pipeline-builder pipeline)))
+
+(defn- add-channel-tracker
+  [pipeline-builder chan-group]
+  (fn [pipeline]
+    (prepend-handler-to-pipeline pipeline "channel-tracker"
+                                (channel-tracking-handler chan-group))
+    (pipeline-builder pipeline)))
```

Comment:
For naming parity, `add-ssl-handler` and `add-channel-tracker` should match. Maybe `add-channel-tracker-handler`?